DemoWebsites/BlazorComponentDemoTestSite/DSD.MSS.Blazor.Components.DemoSite/Pages/_Host.cshtml

```diff
@@ -52,9 +61,54 @@
             $(this).closest("li").toggleClass("active", this.checked);
         });
 
+        $("input").on("change", function () {
+            this.setAttribute(
+                "data-date",
+                moment(this.value, "YYYY-MM-DD")
+                    .format(this.getAttribute("data-date-format"))
+            );
+            console.log("Change to " + this.value);
+        }).trigger("change");
+
+        // jQuery date picker
+        function jqueryDatepicker(id, format, lang) {
+
+            console.log("ID = " + id + " | " + format + " | Lang = " + lang);
+
+            elementRef = document.getElementById(id);
+
+            elementRef.onchange = function () {
+
+                console.log("Value = "+document.getElementById(id).value);
+
+                var event = new Event('change');
+
+                elementRef.dispatchEvent(event);
+
+            };
+
+            if (lang == "fr") {
+
+            } else {
+                lang = "";
+            }
+
+            var options = $.extend({},
+                $.datepicker.regional[lang],
+                {
+                    dateFormat: format, // show date format mm/dd/yy.
+                    changeYear: true,
+                    changeMonth: true
+                }
+            );
+
+            $('input[id$=' + id + ']').datepicker(options);
+        }
+
```

Comment:
Put this in a separate js file for better code orginization.

Reply:
Done